server/graphql/v2/object/Host.js

```diff
@@ -385,6 +385,7 @@ export const Host = new GraphQLObjectType({
             },
             limit: args.limit,
             offset: args.offset,
+            order: [['createdAt', 'DESC']],
```

Comment:
Better to implement it like in `interface/Account.js` virtualCards resolver.

```
      orderBy: {
        type: ChronologicalOrderInput,
        defaultValue: ChronologicalOrderInput.defaultValue,
      },
```

then

```
 order: [[args.orderBy.field, args.orderBy.direction]],
```